packages/@aws-cdk/assertions/README.md

```diff
@@ -129,9 +129,19 @@ assert.hasOutput('*', {
 });
 ```
 
-`findOutputs()` will return a list of outputs that match the `logicalId` and `props`,
+`findOutputs()` will return a set of outputs that match the `logicalId` and `props`,
 and you can use the `'*'` special case as well.
 
+```ts
```

Comment:
I feel like this example is enough to explain the value proposition of the `find*()` APIs without explicitly stating the two outputs `Foo` and `Bar` exist before `assert.findOutputs()`. What do you think?